api/src/main/resources/marquez/db/migration/V52__lineage_events_created_at_indexed.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE lineage_events ADD created_at TIMESTAMP;
```

Comment:
The column will be populated with db server's `utc now() ` to avoid time skews that might happen within different application pods. This column determines the time this event was inserted in the database.

Reply:
Yeah, using `created_at` would be more appropriate!